VultisigApp/VultisigApp/Extensions/StringExtension.swift

```diff
@@ -63,38 +63,16 @@ extension String {
 
 extension String {
     func formatCurrency() -> String {
-        let currency = SettingsCurrency.current
         let decimalPoint = getCurrentDecimalPoint()
-        
-        if currency.usesEuropeanFormat || decimalPoint == "," {
-            return self.replacingOccurrences(of: ",", with: ".")
-        } else {
-            return self.replacingOccurrences(of: ",", with: "").replacingOccurrences(of: ",", with: ".")
-        }
+        return self.replacingOccurrences(of: ",", with: decimalPoint)
     }
     
     func formatCurrencyWithSeparators() -> String {
         guard let number = parseInput() else {
             return self
         }
         
-        let currency = SettingsCurrency.current
-        let decimalPoint = getCurrentDecimalPoint()
-        
-        let outputFormatter = NumberFormatter()
-        outputFormatter.numberStyle = .decimal
-        outputFormatter.minimumFractionDigits = 0
-        outputFormatter.maximumFractionDigits = 8
-        
-        if (currency.usesEuropeanFormat || decimalPoint == ",") {
-            outputFormatter.decimalSeparator = ","
-            outputFormatter.groupingSeparator = "."
-        } else {
-            outputFormatter.decimalSeparator = "."
-            outputFormatter.groupingSeparator = ","
-        }
-        
-        return outputFormatter.string(for: number) ?? self
+        return number.formatToFiat(includeCurrencySymbol: false, useAbbreviation: false)
```

Comment:
I don't think you can replace this one with `formatToFiat` , the one on the left which allow you to keep 8 decimals , while formatToFiat is only 2 decimals

This is me trying to send `0.0001` ETH , it show as 0.00 ETH
![CleanShot 2025-03-13 at 10 55 07@2x](https://github.com/user-attachments/assets/3bd9d4b1-480f-45d3-a09a-a6ed934ae323)